target/linux/ramips/image/mt7621.mk

```diff
@@ -326,6 +326,22 @@ define Device/elecom_wrc-1167ghbk2-s
 endef
 TARGET_DEVICES += elecom_wrc-1167ghbk2-s
 
+define Device/elecom_wrc-1750gs
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 11264k
+  DEVICE_VENDOR := ELECOM
+  DEVICE_MODEL := WRC-1750GS
+  DEVICE_ALT0_VENDOR := ELECOM
+  DEVICE_ALT0_MODEL := WRC-1750GSV
```

Comment:
To be honest, here I'd just have put a 
DEVICE_MODEL := WRC-1750GS(V)
or
DEVICE_MODEL := WRC-1750GS/GSV

IMO that's simple enough to not having to introduce an alternative name.

Reply:
Indeed, I'll update the name to ```WRC-1750GS/GSV``` and drop alternative name.